src/base/filesystemwatcher.cpp

```diff
@@ -28,14 +28,14 @@
 
 #include "filesystemwatcher.h"
 
+#include <QtGlobal>
```

Comment:
This move is correct, as QtGlobal is place just above `Q_OS_MAC`.

Reply:
Ok guys this is confusing, i suppose i took the guidelines too literally. I'll undo all these QtGlobal moving around except this one because you said it's correct although i still don't get why the conditional below makes this a valid move unlike the other cases.

Reply:
`Q_OS_MAC` is defined in QtGlobal so we need to include this header before it's used.